battleship.py

```diff
@@ -204,7 +209,8 @@ def main_game_loop(self):
         
         ## removed make_battle_board from the main game loop to prevent creating new board on every loop instance
         while self.is_running:
-            # print("Attempt #1 at Battleships")
+            sys.stdout.flush()
```

Comment:
What does flush do in this case? I can imagine it does something with the output

Reply:
Just flushes out the current content of the terminal. 

Doesn't actually seem to work so this could actually be removed